### ISSUE TITLE
Tooltip formatter doesn't work

### ISSUE THREAD

**Issue by turorudolf** [pablojim/highcharts-ng#360]
I'm trying to make use of the tooltip formatter callback function but it seems it doesn't work..
I'd do the same as in 
http://jsfiddle.net/gh/get/jquery/1.7.2/highslide-software/highcharts.com/tree/master/samples/highcharts/tooltip/formatter-simple/

Also, my data point objects have some properties other than just x,y coordinates that I'd display in the tooltip too...
Is there an existing way to achive this? If not, are you planning to include these?
Many thanks!

**Comment by dfromero** [comment]
hi @turorudolf are you placing the "options", not directly in the highcharts-ng config object

**Comment by turorudolf** [comment]
hey @dfromero you're right! that was i needed! thnx!

**Comment by jonathanwoahn** [comment]
I'm actually having an issue with the tooltip formatter callback function. I am able to update the tooltip contents correctly, but it's not rendering the html. I've attempted a copy of the object config below:

 vm.userRankingConfig = {
    options: {
      global: {
        useUTC: false
      },
      chart: {
          type: 'bar',
          height: 100
      },
      plotOptions: {
      	bar: {
      		dataLabels: {
      			enabled: true
      		}
      	},
      },
	    tooltip: {
	    	shared: true,
	    	formatter: function()
	    	{
	    		var value = this;
	    		var string = '<div>';
	    		console.log(value);

	    		string += '<h3><strong>' + value.x + '<strong></h3>';

	    		string += '<ul>';
	    		_.forEach(value.points,function(point){
	    			string += '<li><i style="color: ' + point.point.color + ';"></i>'
	    			string += '<span>' + point.series.name + '</span>'
	    			string += '<span>' + point.point.y + '</span>';
	    			string += '</li>';
	    		});

	    		string += '</ul>';

	    		string += '</div>'
	    		
	    		return string;
	    	},
	    	useHtml: true
	    },
      legend: {
        enabled: false
      }
    }
}

**Comment by dfromero** [comment]
@jonathanwoahn do you have a demo (jsfiddle) including test data for this ?

**Comment by jonathanwoahn** [comment]
Here you go:

http://jsfiddle.net/d377711v/

**Comment by dfromero** [comment]
@jonathanwoahn try this http://jsfiddle.net/diegoromero89/o055rr9o/
according to highcharts docs 
**A subset of HTML is supported. The HTML of the tooltip is parsed and converted to SVG, therefore this isn't a complete HTML renderer. The following tabs are supported: b, strong, i, em, br, span. Spans can be styled with a style attribute, but only text-related CSS that is shared with SVG is handled.**

**Comment by jonathanwoahn** [comment]
Good to know on the highcharts doc. Unfortunate that they don't handle more
tags. I think that solution will work - thanks for your help!

**Comment by pablojim** [comment]
Thanks @dfromero !

**Comment by erezcohen** [comment]
@turorudolf - your code is mostly fine and html IS supported.
Checkout http://jsfiddle.net/erez_cohen/d377711v/5/
The only thing I changed was useHtml to useHTML.
(I ran into this "horrible" case myself and spent hours till I turned to their support).

Regarding the citation above from highcharts docs - the "limited html support" is only relevant on useHTML = false (the default). Once using useHTML = true you get a much broader support.
See http://api.highcharts.com/highstock#tooltip.useHTML.
